corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common/module.scss

```diff
@@ -176,15 +176,8 @@ button.clickable-icon {
   }
 }
 
-.module-banner {
-  background-color: $cc-light-warm-accent-hi;
-  color: black;
-  padding-top: 1rem;
-  padding-bottom: 1rem;
-  border: none;
-  margin-top: -1px;
-  text-align: center;
-  font-size: 14px;
+.restore-as-banner {
+  font-size: 11px;
```

Comment:
I wonder if we should just call it `font-size-11` similar to the color classes available.

Reply:
I thought about that - Bootstrap has [font size utilities](https://getbootstrap.com/docs/5.0/utilities/text/#font-size), but they only go down as small as normal sized text, which is too big here.  We could extend that same system a couple clicks longer